internal/core/items/text.rs

```diff
@@ -561,11 +563,15 @@ impl Item for TextInput {
                 self.has_focus.set(true);
                 self.show_cursor(window_adapter);
                 window_adapter.enable_input_method(self.input_type());
+
+                WindowInner::from_pub(window_adapter.window())
+                    .request_open_virtual_keyboard(self.input_type());
```

Comment:
I think input methods and virtual keyboard should use the same interface. So either every time we enable/disable the time we should call request_open/request_close (like you correctly do!), or perhaps we should unify this by removing the enable_input_method bit from the window adapter and instead change the backends to use the new public API.

What do you think?

Reply:
Do you have something already in the public API? To use the public api make sense.

Reply:
No, there's nothing in the public API yet. It would be part of this PR after all :)

Reply:
Ok thank you, let me check this.

Reply:
As discussed on chat, let's keep that change out of this PR.

Reply:
My idea was that the callback on the window adaptor (which was called show_virtual_keyboard  before https://github.com/slint-ui/slint/commit/8cba0622f58890bc36d9d00da3a6c46e5edce166#diff-d31632a76a36dda343d465573660aa82ad42a1ef9413a5641ffb386f67323e58 ) should indeded be used to control the virtual keyboard.  I missed the discussion about it though.